src/components/TextField/TextField.tsx

```diff
@@ -175,22 +181,29 @@ const TextField: VibeComponent<TextFieldProps, unknown> & {
 
     return (
       <div
-        className={classNames("input-component", wrapperClassName, {
-          "input-component--disabled": disabled
+        className={cx(styles.textField, "input-component", wrapperClassName, {
```

Comment:
I think that when names are really not readable it better to do it now, because we will not have another opportunity to do so after finishing the migration. Really prefer to keep this that way. In this case the css module class names was super weird (input component, input component input and more super long names)